lib/karafka/persistence/consumer.rb

```diff
@@ -12,11 +12,16 @@ class Consumer
       PERSISTENCE_SCOPE = :consumers
 
       class << self
-        # @return [Hash] current thread persistence scope hash with all the consumers
+        # @return [Hash] current thread's persistence scope hash with all the consumers
         def all
           # @note This does not need to be threadsafe (Hash) as it is always executed in a
           # current thread context
-          Thread.current[PERSISTENCE_SCOPE] ||= Hash.new { |hash, key| hash[key] = {} }
+          Thread.current[PERSISTENCE_SCOPE] ||= initial_state
+        end
+
+        # Clears current thread's persistence scope
+        def clear
```

Comment:
why would we claer that?

Reply:
I don't like adding methods that are only used in specs. Can there be a scenario that this method will be used in production environment? If not, can we move it to specs?